content/administration/on_premise/source.rst

```diff
@@ -273,18 +273,23 @@ Dependencies
 
          .. tab:: Debian/Ubuntu
 
-            For Debian-based systems, the packages are listed in the `debian/control
-            <{GITHUB_PATH}/debian/control>`_ file of the Odoo sources.
-
             On Debian/Ubuntu, the following commands should install the required packages:
 
             .. code-block:: console
 
-               $ cd /CommunityPath
-               $ sed -n -e '/^Depends:/,/^Pre/ s/ python3-\(.*\),/python3-\1/p' debian/control | sudo xargs apt-get install -y
+               $ cd odoo #CommunityPath
```

Comment:
The *CommunityPath* placeholder is explained above and presented in the `/CommunityPath` format in commands in many other places on this page; let's keep it.
```suggestion
               $ cd /CommunityPath
```
It could maybe be made clearer, but then it should be done for the entire page at once, not just this one command block.